src/runners/helpers/db.py

```diff
@@ -384,14 +399,37 @@ def get_pipes(schema):
     return fetch(f"SHOW PIPES IN {schema}")
 
 
-def create_stage(name, url, prefix, cloud, credentials, file_format, replace=False, comment=''):
+def create_table_from_csv(name, columns, file_path, file_format, ifnotexists=False):
```

Comment:
sweet :-) the name is wrong,  tho -- this is creating the table from the columns arg, not the CSV, error-ing if the table exists, and then loading the CSV into it. maybe `create_table_and_upload_csv` would be a better name?

seems cleaning up the stage after the copy statement would be helpful, as well?

this is the sort of thing that could turn into a command line tool pretty easily with pyfire and a couple of default, e.g. —

    db upload-csv --table=data.somedata --file-path=data.csv

Reply:
I agree with the rename, but I'm not clear what you mean when you say "this is creating the table from the column arg [and] erroring if the table exists"; it'll error if the table a) exists and b) doesn't have the columns that the CSV expects, but the error will be a result of trying to load data in, not table creation. I'm not sure what a better way to handle this would be; you could "CREATE OR REPLACE" the table, but then we lose whatever data was in there originally; I feel like it's better to create if not exists and then if you get an existing table, put the data in there and error if it's wrong.

Reply:
Renamed table and turned on temporary stage for the function.

Not clear what you mean about "erroring if the table exists"; the function should only error if a) the table exists and b) doesn't have the correct columns, in which case it seems safe to assume that it's not actually the right table and dropping it would have been a bummer.

Reply:
Push the change? Got it -- so the second time someone tries to create the table it'll just ignore it.